File: 3oy10 Muhriddin Sobrijonov/2masala.c
#include    <stdio.h>
#include    <stdlib.h>
#include    <time.h>


int main(){
    system("cls");
    int n = 20;
    int son[n];
    srand(time(NULL));
    for(int i = 0; i < n; i++){
        son[i] = rand()%50 +1;
//        printf("%d\t", son[i]);
    }
    FILE *f2 = fopen("2masala.BIN", "wb");
    for(int i = 0; i < n; i++){
        fwrite(&son[i], sizeof(int), 1, f2);
    }
    fclose(f2);
    int  i = 0, m;
    f2 = fopen("2masala.BIN", "rb");
    while (fread(&m, sizeof(int), 1, f2)){
        if(i < m){
            i = m;
        }
    }
    fclose(f2);
    printf("\nEng katta son: %d", i);

    return 0;
}

File: 4oy1 Muhriddin Sobirjonov/2masala.py
import  os
os.system("cls")

a, b, c = map(int, input("3 ta son kiriting:   ").split())

if  a >= b and a > c:
    print("eng katta son  ", a)
elif    b >= a and b >= c:
    print("Eng katta son  ", b)
elif    c >= a and c >= b:
    print("Eng katta son  ", c)